project-loader/src/main/kotlin/ModuleAndModelMatcher.kt

```diff
@@ -13,12 +11,6 @@ class ModuleAndModelMatcher {
         this.excludeModelRegex = regexFromAlternativesOrNull(excludeModels)
     }
 
-    constructor(args: ModelCheckArgs) : this(
```

Comment:
Why was this constructor removed? I would favour having it to simplify the code. 
Otherwise users will have to spell out explicitly the parameter assignments, which may lead to bugs in the future.

Reply:
In a private discussion, it was explained to me that the constructor would need to be duplicated now. Therefore it is removed.